apps/backend/src/websocket/auth.py

```
@@ -0,0 +1,76 @@
+
+logger = logging.getLogger(__name__)
+security = HTTPBearer()
+
+
+class WebSocketAuthenticator:
+    """Handles WebSocket authentication using Bearer tokens"""
+    
+    def __init__(self):
+        self.settings = get_settings()
+    
+    async def authenticate_token(self, token: str) -> Optional[str]:
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)